test/babel/snapshots/index.js.md

```diff
@@ -4,6 +4,43 @@ The actual snapshot is saved in `index.js.snap`.
 
 Generated by [AVA](https://ava.li).
 
+## missing import
+
+> Snapshot 1
+
+    `"use strict";␊
+    ␊
+    var _styleSheet = require("style-sheet");␊
+    ␊
+    /* @jsx createElement */␊
+    var _ref = {␊
+      __cssProp: ["dss10_h28rbs-i0tgik"]␊
+    };␊
+    ␊
+    const ComponentStatic = () => <div css={_ref} />;␊
+    ␊
+    const ComponentDynamic = ({␊
+      margin␊
+    }) => <div css={_styleSheet.StyleSheet.create({␊
+      __cssProp: {␊
+        margin␊
+      }␊
+    })} />;`
+
+> Snapshot 2
+
+    `[style-sheet-group="10"]{}␊
+    .dss10_1jgjtkn-1k19bls{-webkit-filter:blur(10px);filter:blur(10px);}␊
+    .dss10_1eznbv4-6laobc{background-color:hotpink;}␊
+    .dss10_153t8jg-7qvd50{val:10px;}␊
+    .dss10_1avj1fl-7qvd50{font-size:10px;}␊
+    .dss10_u84hpd-dby8y8{padding-top:5px;}␊
+    .dss10_f69c5w-oyp9nw{margin-bottom:10px;}␊
+    .dss10_1buiceq-7qvd50{margin-top:10px;}␊
+    .dss10_h28rbs-i0tgik{color:red;}␊
+    .dss10_j9ctud-1t7uh5u{display:block;}␊
+    .dss10_j9ctud-1kagvzm{display:-webkit-box;display:-moz-box;display:-ms-flexbox;display:-webkit-flex;display:flex;}`
```

Comment:
This is wrong. As I suspected `OrderedCSSStyleSheet` needs a `clear` method (name tbd)